httpx/client.go

```diff
@@ -65,6 +70,28 @@ func FaultTolerance(attempts int, backoff time.Duration) ClientDecorator {
 	}
 }
 
+// InstrumentMetrics returns an adapter that instruments metrics about the request duration
+func InstrumentMetrics(m metrics.Metrics, t ...metrics.Tag) ClientDecorator {
```

Comment:
I think the name should make explicit that we're only instrumenting the request duration. Otherwise you might think that it's also tracking the errors

Reply:
InstrumentRequestDurationMetric would be ok?

Reply:
InstrumentRequestDurationMetric or InstrumentRequestDuration are fine for me

Reply:
done